src/NuGet.Core/NuGet.Common/NuGetEnvironment.cs

```diff
@@ -13,13 +13,12 @@ public static string GetFolderPath(NuGetFolderPath folder)
             switch (folder)
             {
                 case NuGetFolderPath.MachineWideSettingsBaseDirectory:
-                    return Path.Combine(
-                        GetFolderPath(SpecialFolder.CommonApplicationData),
-                        "NuGet");
+                    return GetFolderPath(SpecialFolder.CommonApplicationData);
 
                 case NuGetFolderPath.MachineWideConfigDirectory:
                     return Path.Combine(
                         GetFolderPath(NuGetFolderPath.MachineWideSettingsBaseDirectory),
+                        "nuget",
                         "Config");
 
```

Comment:
I see that you are using the same casing as used in the existing code in Settings.cs. This is good.
However, in this file, NuGetEnvironment.cs, string "NuGet" was used. Hope it does not cause any casing issues in platforms other than Windows.